src/input/espresso.rs

```diff
@@ -532,9 +641,52 @@ impl EpochState {
     }
 
     /// Compute rewards to distribute to each account for a given block.
-    fn reward_distribution(&self, _leader: usize) -> RewardDistribution {
-        // TODO: actually compute rewards
-        vec![]
+    async fn reward_distribution<C: EspressoClient>(
+        &self,
+        leader: usize,
+        _client: &C,
+    ) -> Result<Vec<(Address, ESPTokenAmount)>> {
+        let block_reward_amount = match self.block_reward {
+            Some(reward) => reward,
+            None => {
+                tracing::warn!(
+                    "Block reward not found for epoch={}, skipping reward distribution",
+                    self.number
+                );
+                return Ok(vec![]);
```

Comment:
This seems like it should be an error rather than a warning if it could potentially lead to incorrect results